services/util/util.go

```diff
@@ -80,6 +81,33 @@ func (f optionfunc) apply(opts *cmdOptions) {
 	f(opts)
 }
 
+// OptionsEqual returns true if the results of applying both Options on
```

Comment:
Nit: remove the "on an empty cmdOptions" part of the comment. We don't need to reference the unexported object in the doc comments.

Reply:
Removed. Thanks!